R/config.R

```diff
@@ -668,19 +668,33 @@ windows_cluster <- function(cluster) {
 ## NOTE: Practically supporting old versions (currently 3.3.x and
 ## below) requires work in at least provisionr to get all the packages
 ## built.  See didehpc issue #54
+##
+## Caching for versions, so we don't hit the API excessively
+
+cluster_r_versions <- NULL
```

Comment:
I don't think this will work in the installed package.  Typically we do this with a package _environment_ so something like

```
cache <- new.env(parent.env = emptyenv())
```

then assigning into that

```
if (is.null(cache$r_versions) || refresh) {
   ...
   cache$r_versions <- whatever
}
```

I don't think we need the time-based caching here - once per session should be ample

Reply:
Done!